test/e2e/service.go

```diff
@@ -1151,6 +1171,31 @@ func testReachable(ip string, port int) bool {
 	return testReachableInTime(ip, port, podStartTimeout)
 }
 
+func testNetcatReachable(ip string, port int) {
+	con, err := net.Dial("udp", ip+":"+string(port))
```

Comment:
The test was added in before testReachableInTIme polled and I never added it.  Sure, I can add it.